com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/BundleHelper.java

```diff
@@ -608,11 +608,11 @@ public void copyIosIcons() throws IOException
         BufferedImage largestIconImage = getFallbackIconImage("ios", iconPropNames);
 
         // Copy game.project specified icons
-        genIcon(largestIconImage, appDir, "ios",   "app_icon_76x76",      "Icon-76.png",  76);
-        genIcon(largestIconImage, appDir, "ios", "app_icon_152x152",   "Icon-76@2x.png", 152);
-        genIcon(largestIconImage, appDir, "ios", "app_icon_120x120",   "Icon-60@2x.png", 120);
-        genIcon(largestIconImage, appDir, "ios", "app_icon_180x180",   "Icon-60@3x.png", 180);
-        genIcon(largestIconImage, appDir, "ios", "app_icon_167x167", "Icon-83.5@2x.png", 167);
+        genIcon(largestIconImage, appDir, "ios", "app_icon_120x120",          "AppIcon60x60@2x.png", 120);
```

Comment:
Renaming these to be the same as in a default Xcode project. This makes it easier to compare a Defold bundle with and Xcode bundle when searching for discrepancies.